plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -967,7 +974,7 @@ $.extend(DataTable.prototype, UIControl.prototype, {
             }
         });
 
-        const $dataTable = $searchInput.parents('.dataTable').first();
+        var $dataTable = $searchInput.parents('.dataTable').first();
```

Comment:
As this file is possibly loaded in older browsers, and was missed last time when implementing the second search field, I've updated those now.